tests/BUILD

```diff
@@ -24,7 +24,6 @@ load("//tools:cluster_test_rule.bzl", "typedb_cluster_py_test")
 load("@vaticle_bazel_distribution//artifact:rules.bzl", "artifact_extractor")
```

Comment:
We restructured the test package a little, introducing a specific BUILD file for the integration tests only which did not exist before